pkg/controller/scyllacluster/resource.go

```diff
@@ -916,13 +992,26 @@ func agentContainer(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster) corev1.Conta
 		Name:            "scylla-manager-agent",
 		Image:           agentImageForCluster(c),
 		ImagePullPolicy: corev1.PullIfNotPresent,
-		Args: []string{
-			"-c",
-			naming.ScyllaAgentConfigDefaultFile,
-			"-c",
-			path.Join(naming.ScyllaAgentConfigDirName, naming.ScyllaAgentConfigFileName),
+		// There is no point in starting scylla-manager before ScyllaDB is tuned and ignited. The manager agent fails after 60 attempts and hits backoff unnecessarily.
+		Command: []string{
+			"/usr/bin/bash",
+			"-euEo",
+			"pipefail",
+			"-O",
+			"inherit_errexit",
 			"-c",
-			path.Join(naming.ScyllaAgentConfigDirName, naming.ScyllaAgentAuthTokenFileName),
+			strings.TrimSpace(`
+printf '{"L":"INFO","T":"%s","M":"Waiting for /mnt/shared/ignition.done"}\n' "$( date -u '+%Y-%m-%dT%H:%M:%S,%3NZ' )" > /dev/stderr
+until [[ -f "/mnt/shared/ignition.done" ]]; do
+  sleep 1;
+done
+printf '{"L":"INFO","T":"%s","M":"Ignited. Starting ScyllaDB Manager"}\n' "$( date -u '+%Y-%m-%dT%H:%M:%S,%3NZ' )" > /dev/stderr
```

Comment:
```suggestion
printf '{"L":"INFO","T":"%s","M":"Ignited. Starting ScyllaDB Manager Agent"}\n' "$( date -u '+%Y-%m-%dT%H:%M:%S,%3NZ' )" > /dev/stderr
```

It's also still called Scylla Manager, but that's a nit

Reply:
done

Reply:
probably half way but their docs says ScyllaDB Manager https://manager.docs.scylladb.com/branch-3.3/index.html
fixed the agent part